gulpfile.js

```diff
@@ -353,8 +352,12 @@ function deployCesium(bucketName, uploadDirectory, cacheControl) {
     var totalFiles = 0;
     var uploaded = 0;
     var skipped = 0;
+    var errors = [];
 
     return listAll(s3, bucketName, uploadDirectory, existingBlobs)
+    .catch(function(error) {
```

Comment:
Before this catch block was inside the `listAll` function itself, but I moved it out. Should I omit the catch block entirely?